src/Providers/EventDataSynchronizerServiceProvider.php

```diff
@@ -109,6 +111,7 @@ class EventDataSynchronizerServiceProvider extends EventServiceProvider
             CustomerIoSyncEventListener::class . '@handleSubscriptionRenewed',
             IntercomSyncEventListener::class . '@handleSubscriptionRenewed',
             HelpScoutEventListener::class . '@handleSubscriptionRenewed',
+            ImpactEventListener::class . '@handleSubscriptionRenewed',
```

Comment:
@mirceamartsoft - We no longer want to track renewal events. Please remove this listener for now. We may need to enable it in the future so the handler code can stay.

Reply:
done